node/libs/roles/src/validator/messages/consensus.rs

```diff
@@ -227,6 +181,32 @@ pub struct PrepareQC {
 }
 
 impl PrepareQC {
+    /// View of the QC.
+    pub fn view(&self) -> ViewNumber {
+        self.map
+            .keys()
+            .map(|k| k.view)
+            .next()
+            .unwrap_or(ViewNumber(0))
+    }
+
+    /// Add a validator's signed message.
+    /// * `signed_message` - A signed message.
+    /// * `validator_index` - A tuple containing the index of the validator and the total size of the set.
+    pub fn add(
+        &mut self,
+        signed_message: &Signed<ReplicaPrepare>,
+        validator_index: (usize, usize),
```

Comment:
Why not &ValidatorSet? `(usize,usize)` is a very poor choice for an interface.

Reply:
+1

Reply:
Was I overly excited to finally use built-in tuples syntax? 😁

Fixed.